tests/__sf_section_golang.test.fish

```diff
@@ -3,7 +3,7 @@ source $DIRNAME/mock.fish
 
 function setup
 	spacefish_test_setup
-	mock go 0 "echo \"go version go1.10.3 darwin/amd64\""
+	mock go version 0 "echo \"go version go1.10.3 darwin/amd64\""
```

Comment:
The same here, did you get variable passing working?
How does it work? :blush:

Reply:
Yep! It's documented here:
https://github.com/matchai/spacefish/blob/74dec4a89b946ee9aa36beca8ac1650a9e3efe1f/tests/mock.fish#L1-L11

Reply:
I'll update it to add usage examples.

Reply:
Oh!
> ('*' defines a fallback for all arguments) 

Is this the purpose of... `\*`? Should it also be documented that it needs escaping?

Reply:
Yeah, I could specify that here. Before I do, do you have any issues with the API as it is? Would something be better suited than `\*`?

Reply:
Not that I can think of! Just needs a tiny bit of explaination.
I've usually seen something like `**args` or `$argv` or something. At first glance a wildcard makes me think of file globbing. I don't mind that much though, heh.

In the documentation, I'm not exactly a fan of using `$` for the example command. It implies a prompt or interactive use, seems a little odd. I'd also like to ask for a single blank line between usage information and the code. :yum:

Reply:
@matchai

Reply:
Yep! Totally fair. I'll get rid of the `$`s.